configs/components/windows_puppet.json

```diff
@@ -1,4 +1,4 @@
 {
   "url": "git://github.delivery.puppetlabs.net/puppetlabs-puppet_for_the_win.git",
-  "ref": "origin/master"
+  "ref": "refs/tags/4.5.1"
```

Comment:
cc/ @Iristyle there were no changes to the master branch since that 4.5.1 tag was put in. I'm happy to add a 4.5.2 tag to the same sha if that helps